mods/mc_teacher/maps/MKRF512_aspect.conf

```
@@ -3,11 +3,14 @@ name = Malcolm Knapp Research Forest - aspect
 spawn_pos_x = 279
 spawn_pos_y = 63
 spawn_pos_z = 161
-schematic_size_x = 513
+schematic_size_x = 512
 schematic_size_y = 175
-schematic_size_z = 513
-license = CC BY-SA 4.0
+schematic_size_z = 512
+offset_x = -1
+offset_y = -1

```

There is a 1-block high air gap on the bottom-most layer of the realm
```suggestion
offset_y = -2
```